File: 01.JS-Basic/По сложни проверки Упражнение/Cinema.js
function cinema(input) {

    let type = input[0]
    let r = Number(input[1])
    let c = Number(input[2])

    let capacity = r * c

    if (type === "Premiere") {
        console.log((capacity * 12).toFixed(2))
    }
    else if (type === "Normal") {
        console.log((capacity * 7.5).toFixed(2))
    }
    else if (type === "Discount") {
        console.log((capacity * 5).toFixed(2))
    }
}
cinema(["Premiere", "10", "12"])

File: JAVA-Basic/JavaBasic/src/ConditionalStatementsAdvanced/Exercise/Cinema.java
package ConditionalStatementsAdvanced.Exercise;

import java.util.Scanner;

public class Cinema {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        String type = scan.nextLine();
        int r = Integer.parseInt(scan.nextLine());
        int c = Integer.parseInt(scan.nextLine());
        double totalPlaces = r * c;

        switch (type) {
            case "Premiere":
                System.out.printf("%.2f leva", totalPlaces * 12);
                break;
            case "Normal":
                System.out.printf("%.2f leva", totalPlaces * 7.5);
                break;
            case "Discount":
                System.out.printf("%.2f leva", totalPlaces * 5);
                break;
        }
    }
}
